extensions/claude-throne/webview/main.js

```
@@ -165,66 +504,202 @@
       case 'keyStored':
         handleKeyStored(message.payload);
         break;
+      case 'anthropicKeyStored':
+        handleAnthropicKeyStored(message.payload);
+        break;
       case 'proxyError':
-        showError(message.payload);
+        // Comment 1: Handle structured error payload
+        const proxyErrorPayload = typeof message.payload === 'string' 
+          ? { provider: state.provider, error: message.payload, errorType: 'generic' }
+          : message.payload;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)